spec/spec_helper.rb

```diff
@@ -11,10 +11,20 @@
 
 require "vcr"
 
+TEST_URL = "http://localhost:3000"
```

Comment:
I would have named it ROOT_TEST_URL or APIV3_URL or APIV3_ROOT, something more than the non-descriptive "test url"